packages/app/src/screens/SettingsScreen.tsx

```diff
@@ -1,14 +1,59 @@
-import React from 'react';
-import { View } from 'react-native';
+import React, { useState } from 'react';
+import { StyleSheet, SafeAreaView } from 'react-native';
 
-import { Text } from 'ui';
+import { Text, Stack, ListContainer, Divider, SwitchButton } from 'ui';
+import { colors } from 'design-token';
+import Profile from 'components/Profile';
+import { icon_arrowRight } from 'icons';
+import { SvgXml } from 'react-native-svg';
 
 const SettingScreen = () => {
+  const [arrivalNoti, setArrivalNoti] = useState(true);
+  const [weatherNoti, setWeatherNoti] = useState(true);
   return (
-    <View>
-      <Text>SettingScreen</Text>
-    </View>
+    <SafeAreaView style={styles.wrapper}>
+      <Profile />
```

Comment:
```suggestion
    <View style={styles.wrapper}>
      <ScrollView>
        <Profile />
```

TabNavigation을 사용하고 header가 있는 경우에는 굳이 SafeAreaView를 사용하지 않아도 될 것같아요!
그리고 설정 화면의 경우 경험상 리스트가 꽉 차지 않아도 스크롤이 가능한 경우가 더 많은 것같아서, ScrollView를 넣는 건 어떤가요?ㅎㅎ

Reply:
ScrollView로 하는게 맞는 것 같아요!! ㅎㅎ 그리고 header 있는 경우에도 다 SafeAreaView로 일단 해놨었는데, 일단 피그마 화면 토대로 바꿔놨습니다!